API/CLI.md

```diff
@@ -15,6 +15,20 @@ The best place to go next is [the Tessel 2 start experience](http://tessel.io/t2
 ## Updating Tessel 2 On-board OS/Firmware
 
 ### How do I know if I need to update my T2?
+t2-cli and t2-firmware are separately versioned.
+
+* For updating t2-cli:
+
+  If the computer running the t2-cli is connected to the internet, the CLI checks whether it is up to date or not. If it is not, the CLI will provide the update instructions using a notifier to the user (as shown below). It is to be updated with a global npm re-install of the CLI.
+
+  _The CLI only checks once in a day ensuring that the CLI functioning does not slow down._
+
+  <img src="/images/CLI_Update.png" width="400">
+
+* For updating t2-firmware:
+
+  The user does not get update information for t2-firmware from the CLI. The most convenient and fastest way to find is by running the `t2 update` command.
```

Comment:
Missing a phrase, "way to find is" should be "way to find out if a firmware update is needed is"